src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -39,59 +44,105 @@
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static com.exactpro.th2.common.schema.message.FilterFunction.DEFAULT_FILTER_FUNCTION;
+import static com.exactpro.th2.common.metrics.CommonMetrics.DEFAULT_BUCKETS;
+import static java.util.Objects.requireNonNull;
 
 public abstract class AbstractRabbitSubscriber<T> implements MessageSubscriber<T> {
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractRabbitSubscriber.class);
 
+    private static final Counter MESSAGE_SIZE_SUBSCRIBE_BYTES = Counter.build()
+            .name("th2_rabbitmq_message_size_subscribe_bytes")
+            .labelNames(TH2_PIN_LABEL, TH2_TYPE_LABEL, QUEUE_LABEL)
+            .help("Quantity of received bytes from RabbitMQ, includes bytes of messages dropped after filters. " +
+                    "For dropped please see 'th2_message_dropped_subscribe_total' and 'th2_message_group_dropped_subscribe_total'. " +
+                    "The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content")
```

Comment:
```suggestion
                    "The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content")
```